components/atoms/ToggleGroup/toggle-group.tsx

```diff
@@ -10,7 +10,7 @@ interface ToggleGroupProps {
     defaultSelection?: string | number,
     /** Callback function that is called when the user selects an option, and the position of the selected element is passed into as string, starting with '0'. */
     handleChange?: (value: string) => void,
-    className: string
+    className?: string
```

Comment:
any reason this was required and is now optional?

Reply:
No real reason for being required, just forgot to make it optional when I first made it.